test/cpu_fusion.cpp

```diff
@@ -1348,10 +1348,13 @@ TEST(cpu_fusion, rnn_fusion_from_json_model)
     stringstream ss(json_string);
     shared_ptr<Function> func = ngraph::deserialize(ss);
     pass_manager.run_passes(func);
-    const size_t NUM_STEPS = 10;
+    // Replace variable NUM_STEP by its value to work around MSVC bug: 'NUM_STEPS' cannot be implicitly 
+    // captured because no default capture mode has been specified
+    //const size_t NUM_STEPS = 10;
```

Comment:
specify default capture mode (=) instead of commenting out NUM_STEPS.

Reply:
Fixed.